contrib/adaptive-compression/adapt.c

```diff
@@ -42,6 +42,8 @@ static size_t g_streamedSize = 0;
 static unsigned g_useProgressBar = 1;
 static UTIL_freq_t g_ticksPerSecond;
 static unsigned g_forceCompressionLevel = 0;
+static unsigned g_minCLevel = 1;
+static unsigned g_maxCLevel = 22;
```

Comment:
minor : you could also use `ZSTD_maxCLevel()` to get `g_maxCLevel` : https://github.com/facebook/zstd/blob/dev/lib/zstd.h#L134